README.md

```diff
@@ -155,19 +155,21 @@ The `trunk serve` command accepts two proxy related flags.
 
 `--proxy-backend` specifies the URL of the backend server to which requests should be proxied. The URI segment of the given URL will be used as the path on the Trunk server to handle proxy requests. E.G., `trunk serve --proxy-backend=http://localhost:9000/api/` will proxy any requests received on the path `/api/` to the server listening at `http://localhost:9000/api/`. Further path segments or query parameters will be seamlessly passed along.
 
-`--proxy-rewrite` specifies an alternative URI on which the Trunk server is to listen for proxy requests. Any requests received on the given URI will be rewritten to match the URI of the proxy backend, effectively stripping the rewrite prefix. E.G., `trunk serve --proxy-backend=http://localhost:9000/ --proxy-rewrite=/api/` will proxy any requests received on `/api/` over to `http://localhost:9000/` with the `/api/` prefix stripped from the request, while everything following the `/api/` prefix will be left unchanged.
+`--proxy-path` specifies the path on which the Trunk server is to listen for proxy requests. Any requests received on the given URI will be routed to the proxy backend. E.G., `trunk serve --proxy-backend=http://localhost:9000/api --proxy-path=/api/` will proxy any requests received on `/api/` over to `http://localhost:9000/api`.
+
```

Comment:
I think the naming update is good, it is a bit more intuitive, though it is a breaking change. We'll need to add a line to the changelog describing this breaking change, and how folks can update their workflows to account for the change. Namely, we should have:
- a line describing the fact that `--proxy-rewrite` has been rename to `--proxy-path`.
- a line describing what will need to be updated in the Trunk.toml.

We don't have anything like this currently in the changelog, but we should basically add a heading just above the most recent `0.7.x` entry, and call it something like `## Upgrading from 0.7`. The lines I mentioned above should go under this heading.

Reply:
I added that section. Let me know if there's something that needs to be changed.